junit-runtime/src/main/scala/org/junit/runners/BlockJUnit4ClassRunner.scala

```diff
@@ -0,0 +1,20 @@
+package org.junit.runners
+
+import org.junit.runner.Description
+import org.junit.runner.manipulation.{Filter, Sorter}
+import org.junit.runner.notification.RunNotifier
+import org.junit.runners.model.FrameworkMethod
+
+class BlockJUnit4ClassRunner(testClass: Class[_]) extends ParentRunner[FrameworkMethod](testClass) {
+  // Dummy for classOf[...]
```

Comment:
A comment explaining why it's OK to have only dummies, and why we need them in the first place, would be good.